application/src/test/java/eu/europa/ec/fisheries/uvms/activity/rest/BaseActivityArquillianTest.java

```diff
@@ -136,7 +135,7 @@ protected void setUp() throws NamingException, ServiceException, JAXBException,
     }
 
     @Transactional
-    private void populateFluxTestData() throws IOException, JAXBException, ServiceException {
+    public void populateFluxTestData() throws IOException, JAXBException, ServiceException {
```

Comment:
Why is this now public?

Reply:
Interceptors do not work for private methods